docs/visualization/plotting/customizing-figures.ipynb

```diff
@@ -6,6 +6,16 @@
    "source": [
     "# Customizing figures\n",
     "\n",
+    "<div class=\"alert alert-warning\">\n",
+    "\n",
+    "*Deprecated*\n",
+    "\n",
+    "This particular guide uses the old plotting backend which is now deprecated and will be removed in the future.\n",
+    "See [Migrating to the New Backend](../plotting-overview.ipynb#Migrating-to-the-New-Backend) for general migration instructions.\n",
+    "And see the documentation of [Plopp](https://scipp.github.io/plopp/index.html) for how to customize figures with the new backend.\n",
```

Comment:
I don't think one should start sentences with "And"?

Reply:
That is a false 'rule': https://www.thesaurus.com/e/grammar/starting-a-sentence-with-and/

Reply:
Ok, but it sounds really odd when followed by a verb.

Reply:
I should add a page in plopp docs on how to make custom figures, then we could link to that